Documentation/Articles/Xcode-Integration.md

```diff
@@ -4,9 +4,7 @@ To integrate SwiftGen directly in an Xcode project, and make it generate the fil
 
 ## Adding SwiftGen as a Run Script Phase
 
-You can read [Apple's dedicated documentation which explains how to add a Run Script Phase (with illustrations) here](https://developer.apple.com/library/ios/recipes/xcode_help-project_editor/Articles/AddingaRunScriptBuildPhase.html)
-
-In short, it consists of these steps:
```

Comment:
I was almost sure that people rejoiced on Twitter that there was now a proper documentation page at Apple about Build Phases (like, since they added xcfilelist, they decided it was finally time to create a proper documentation page about Build Phases), so would be nice to link to it, but can't find it anymore…

Reply:
Ah, it seems it's only available in the built-in Xcode 10 help, but not on a web page

Help Menu > Xcode Help > search "run a shell script"

Reply:
Ha, here it is: https://help.apple.com/xcode/mac/10.0/index.html?localePath=en.lproj#/devc8c930575

(h/t @ZevEisenberg)

Maybe still worth linking to it then?